keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -149,3 +159,284 @@ On downgrade, the feature gate will simply be disabled, and as long as `kube-pro
 ### Version Skew Strategy
 
 Version skew from the control plane to `kube-proxy` should be trivial since `kube-proxy` will simply ignore the `ipMode` field.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: LoadBalancerIPMode
+  - Components depending on the feature gate: kube-proxy, kube-apiserver
+
+###### Does enabling the feature change any default behavior?
+
+No.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes.
```

Comment:
nit: Yes, by disabling the feature gate.

Probably worth mentioning that to get user-visible impact, disabling it in kube-proxy is enough, right?

Reply:
Yes. Thanks for your advice.